website/static/js/filepage/index.js

```diff
@@ -194,6 +195,49 @@ var FileViewPage = {
                 ], '.')
             ]);
         }
+        var sharePopover =  {view: function() { return m('button.btn.btn-sm.btn-primary.file-share', {onclick: function () {
+                var link = $('iframe').attr('src');
+                var height = $('iframe').attr('height');
+                link = link.substring(0, link.indexOf('download') + 8) + '%26mode=render';
+                var url = link.substring(0, link.indexOf('render'));
+                m.render(document.getElementById('popOver'), [
+                    m('ul.nav.nav-tabs.nav-justified', [
+                        m('li.active', m('a[href="#share"][data-toggle="tab"]', 'Share')),
+                        m('li', m('a[href="#embed"][data-toggle="tab"]', 'Embed'))
+                    ]), m('br'),
+                    m('.tab-content', [
+                        m('.tab-pane.fade.in.active#share', m('.input-group', [
+                            m('span.input-group-btn', m('button#copyBtn.btn.btn-default.btn-md[type="button"][style="height: 34px"][data-clipboard-text="' + link + '"]', m('.fa.fa-copy'))),
```

Comment:
This is a number that I believe came from the hashes pr. The no button class had the same height as the text-area.